framework/src/modules/chain/submodules/loader.js

```diff
@@ -473,13 +473,12 @@ __private.loadBlockChain = function() {
 		async.series(
 			{
 				resetMemTables(seriesCb) {
-					library.logic.account.resetMemTables(err => {
-						if (err) {
-							throw err;
-						} else {
-							return setImmediate(seriesCb);
-						}
-					});
+					library.storage.entities.Account.resetMemTables()
+						.then(() => seriesCb())
```

Comment:
`.then(seriesCb)` ?

Reply:
this should be better because if then returns something it will be treated as error